serpy/serializer.py

```diff
@@ -83,16 +123,19 @@ class FooSerializer(Serializer):
     """
     #: The default getter used if :meth:`Field.as_getter` returns None.
     default_getter = operator.attrgetter
+    default_setter = attrsetter
 
-    def __init__(self, obj=None, many=False, **kwargs):
+    def __init__(self, obj=None, data=None, many=False, **kwargs):
```

Comment:
yeah, I like your proposal below about using `instance` for serialization and `data` for deserialization, lets go with that